sky/packages/sky/lib/src/material/theme_data.dart

```diff
@@ -2,10 +2,10 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:ui';
+import 'dart:ui' show Color;
 
-import 'package:flutter/src/material/typography.dart';
-import 'package:flutter/src/material/colors.dart';
+import 'typography.dart';
+import 'colors.dart';
```

Comment:
actually I guess not. It's not technically part of colors.dart's API, it's an implementation detail.